app-misc/sl/sl-5.02.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="sophisticated graphical program which corrects your miss typing"
+HOMEPAGE="http://www.tkl.iis.u-tokyo.ac.jp/~toyoda/index_e.html https://github.com/mtoyoda/sl/"
+SRC_URI="https://github.com/mtoyoda/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="freedist"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~x86-interix ~amd64-linux ~x86-linux ~ppc-macos"
+
+DEPEND="sys-libs/ncurses:0"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	eapply_user
+	sed -e "s/-lncurses/\`$(tc-getPKG_CONFIG) --libs ncurses\`/" -i Makefile
+}
+
+src_install() {
+	dobin sl
+	dodoc README.md
```

Comment:
Addressed all of the above. For this one i also introduced IUSE l10n_ja, not sure how clean the "rm" is to exclude the japanese README from einstalldocs in case of !l10n_ja.